LessonOneHomework/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace LessonOneHomework
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool continueLoop = true;
+            while(continueLoop)
+            {
+                (string firstName, string surname) = GetUserName();
+                Console.Write("What is your age? ");
+                int age = int.Parse(Console.ReadLine());
+                (double height, double weight) = GetHeightandWeight();
+                double bmi = CalculateBMI(height, weight);
+                Console.WriteLine($"{firstName} {surname} is {age} years old, his weight is {weight} and his height is {height}.");
+                Console.WriteLine($"Their BMI is {bmi}.");
+                Console.Write("Another person? (y/n)");
+                string looper = Console.ReadLine();
+                continueLoop = looper.ToLower() == "n" ? false : true;
+            }
+        }
+
+        private static (string, string) GetUserName()
```

Comment:
"Get*" may sound a bit vague, you might as well name it `PromptUsername()` just to be clear about what this does